lib/bencher_json/src/project/measure/defs.rs

```diff
@@ -0,0 +1,128 @@
+use crate::JsonNewMeasure;
+use bencher_valid::NameId;
+
+pub trait MeasureDefinition {
+    const NAME_STR: &'static str;
+    const SLUG_STR: &'static str;
+    const UNITS_STR: &'static str;
+
+    fn name_id() -> NameId {
+        Self::SLUG_STR
+            .parse()
+            .expect("Failed to parse measure slug.")
+    }
+
+    fn json_new() -> JsonNewMeasure {
```

Comment:
Pedantic, but I would prefer the verb first, so `new_json` instead of `json_new`.

Reply:
Sure. I was following the naming of the existing `JsonNewMeasure` type.